packages/aksara-ui-core/src/components/side-sheet/components/SideSheetContent.tsx

```diff
@@ -1,9 +1,16 @@
 import * as React from 'react';
-import { DialogContent, DialogContentProps } from '../../dialog';
+import { Box } from '../../../foundations';
+import { DialogContentProps } from '../../dialog';
```

Comment:
```suggestion
import { Box, BoxProps } from '../../../foundations';
```

Pake `BoxProps` aja.